src/components/picker/picker.tsx

```diff
@@ -252,94 +248,77 @@ export class Picker {
      */
     private renderDropdown() {
```

Comment:
This method has a bit too much responsibility so I'm breaking it up since we are poking around in it anyway :slightly_smiling_face: